src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -692,5 +692,153 @@ public static void SyndicationFeed_Rss_WrongDateFormat()
             // *** ASSERT *** \\
             Assert.True(!res.LastUpdatedTime.Equals(new DateTimeOffset()));
         }
+        
+        [Fact]
+        [ActiveIssue(24571)]
+        public static void AtomEntryPositiveTest()
+        {
+            string filePath = @"brief-entry-noerror.xml";
+            string serializeFilePath = Path.GetTempFileName();
+            
+            try
+            {
+                XmlReader reader = XmlReader.Create(filePath , new XmlReaderSettings() {Async=true});
+                Task<SyndicationItem> task = SyndicationItem.LoadAsync(reader);
+                Task.WhenAll(task);
+                SyndicationItem feedObjct = task.Result;
+
+                XmlWriter writer = XmlWriter.Create(serializeFilePath);
+                Atom10ItemFormatter f = new Atom10ItemFormatter(feedObjct);
+                f.WriteToAsync(writer).GetAwaiter().GetResult();
+                writer.Close();
+
+                // compare file filePath and serializeFilePath
+                XmlDiff diff = new XmlDiff();
+                Assert.True(diff.Compare(filePath, serializeFilePath));
+            }
+            finally
+            {
+                File.Delete(serializeFilePath);
+            }
+        }
+
+        [Fact]
+        [ActiveIssue(24572)]
+        public static void AtomEntryPositiveTest_write()
+        {
+            SyndicationItem item1 = new SyndicationItem("SyndicationFeed released for .net Core", "A lot of text describing the release of .net core feature", new Uri("http://Contoso.com/news/path"));
+            string serializeFilePath = Path.GetTempFileName();
+
+            try
+            {
+                XmlWriter writer = XmlWriter.Create(serializeFilePath, new XmlWriterSettings() { Async = true });
+
+                Atom10ItemFormatter f = new Atom10ItemFormatter(item1);
+                Task task = f.WriteToAsync(writer);
+                Task.WhenAll(task);
+                writer.Close();
+
+                //TODO:
+                // Check file content
```

Comment:
Can you fix the TODO?

Reply:
It is blocking due to issue #24571.

Reply:
We can add the implementation and mark the test with both issues.

Reply:
I have add a file to compare the result.